src/Microsoft.AspNetCore.Sockets.Http/Internal/Transports/WebSocketsTransport.cs

```diff
@@ -66,69 +67,51 @@ public WebSocketsTransport(WebSocketOptions options, IChannelConnection<byte[]>
                 receiving,
                 sending);
 
-            // What happened?
+            var failed = false;
+            var task = Task.CompletedTask;
             if (trigger == receiving)
             {
                 if (receiving.IsCanceled || receiving.IsFaulted)
                 {
-                    // The receiver faulted or cancelled. This means the socket is probably broken. Abort the socket and propagate the exception
+                    // The receiver faulted or canceled. This means the socket is probably broken. Abort the socket and propagate the exception
                     receiving.GetAwaiter().GetResult();
```

Comment:
When we are in this branch - do we ever await the `sending` task? What if it throws? Also, how do we make sure the task (and therefore the send loop ever) finishes